README.md

```diff
@@ -0,0 +1,257 @@
+# Intermittent Energy Importers
+
+Powers the analytics backend of [intermittent.energy/](https://intermittent.energy/), providing deep insights into global electricity markets and renewable energy integration.
+
+A high-performance Ruby application for collecting, processing, and analyzing real-time power grid data worldwide. Built on PostgreSQL with [TimescaleDB](https://github.com/timescale/timescaledb/) for lightning-fast time series processing.
+
+Follow [@IntermittentNRG](https://x.com/IntermittentNRG) for updates.
+
+## Data Sources
+
+🌎 **Americas**
+- 🇺🇸 ⚡ [CAISO (California Independent System Operator)](lib/caiso.rb)
+- 🇺🇸 🔋 [EIA (U.S. Energy Information Administration)](lib/eia.rb)
+- 🇺🇸 💡 [ERCOT (Electric Reliability Council of Texas)](lib/ercot.rb)
+- 🇺🇸 🔌 [NYISO (New York Independent System Operator)](lib/nyiso.rb)
+- 🇨🇦 ⚡ [AESO (Alberta Electric System Operator)](lib/aeso.rb)
+- 🇨🇦 🔋 [HYDRO-QUÉBEC](lib/hydro_quebec.rb)
+- 🇨🇦 💡 [IESO (Independent Electricity System Operator - Ontario)](lib/ieso.rb)
+- 🇨🇦 🔌 [NS Power (Nova Scotia Power)](lib/ns_power.rb)
+- 🇧🇷 ⚡ [ANEEL (Brazilian Electricity Regulatory Agency)](lib/aneel.rb)
+- 🇧🇷 🔋 [ONS (National System Operator - Brazil)](lib/ons.rb)
+- 🇦🇷 💡 [CAMMESA (Wholesale Electricity Market Administrator - Argentina)](lib/cammesa.rb)
+
+🌍 **Europe**
+- 🇪🇺 ⚡ [ENTSOE (European Network of Transmission System Operators for Electricity)](lib/entsoe.rb)
+- 🇬🇧 🔋 [ELEXON (GB Electricity Market)](lib/elexon.rb)
+- 🇬🇧 💡 [National Grid ESO (Great Britain)](lib/national_grid.rb)
+- 🇮🇪 🔌 [EIRGRID (Ireland's Electricity Grid Operator)](lib/eirgrid.rb)
+- 🇪🇸 ⚡ [REE (Red Eléctrica de España)](lib/ree.rb)
+- 🇸🇪 🔋 [SVK (Svenska Kraftnät - Sweden)](lib/svk.rb)
+- 🇪🇺 💡 [NORDPOOL (Nordic Power Exchange)](lib/nordpool.rb)
+
+🌏 **Asia-Pacific**
+- 🇦🇺 ⚡ [AEMO (Australian Energy Market Operator)](lib/aemo.rb)
+  - 🔌 [AEMO NEM](lib/aemo_nem.rb) - National Electricity Market
+  - 🔋 [AEMO WEM](lib/aemo_wem.rb) - Western Australia
+  - 💡 [AEMO NEM Archive](lib/aemo_nem_archive.rb) - Historical NEM data
+- 🇦🇺 🔌 [OpenNEM (Australian National Electricity Market Data)](lib/opennem.rb)
```

Comment:
OpenNEM gets their data from AEMO which I now also do. Their historical data doesn't include rooftop solar which I reported on their github, don't think it got fixed. It is useful to keep to compare results tho.